operators/pkg/tenant-controller/tenant_controller.go

```diff
@@ -46,6 +47,21 @@ import (
 const (
 	// NoWorkspacesLabel -> label to be set (to true) when no workspaces are associated to the tenant.
 	NoWorkspacesLabel = "crownlabs.polito.it/no-workspaces"
+	// PVC name
+	UserPvcName = "user-pvc"
+	// PVC secret name
+	PvcSecretName = "user-pvc-secret"
+	// Server key in PVC secret
+	SecretServerNameKey = "server-name"
+	// Server key in PVC secret
+	SecretServerIPKey = "server-ip"
+	// Path key in PVC secret
+	SecretPathKey = "path"
+	// PVC size
+	UserPvcSize = 1 * 1024 * 1024 * 1024 // 1 Gi
```

Comment:
There should be a way to define this using the resource type, if not, I'd still call it like UserPvcSizeBytes. Also, it should be better to make this customizable in some way, would keep this comment open for further discussion.

Reply:
Ok, for the moment I only changed the variable name from `UserPvcSize` to `UserPvcSizeBytes`